libs/ngx-sherfire/src/lib/firestore/firebase-firestore.service.ts

```diff
@@ -1,85 +0,0 @@
-import { inject, Injectable, NgZone } from '@angular/core';
-import firebase from 'firebase/app';
-import 'firebase/firestore';
-import { FirebaseApp } from '../app';
-import { FirebaseConfig } from '../firebase-config.service';
-import { NgxSherfireModule } from '../ngx-sherfire.module';
-import {
-    AugmentedCollectionReference,
-    AugmentedDocumentReference,
-    AugmentedDocumentSnapshot,
-    AugmentedFirestore,
-    AugmentedQuery,
-    AugmentedQueryDocumentSnapshot,
-    AugmentedTransaction,
-    augmentFirestorePrototypes,
-} from './augment-prototypes';
-
-export interface FirebaseFirestore extends AugmentedFirestore {}
-
-@Injectable({
-    providedIn: NgxSherfireModule,
-    useFactory: firebaseFirestoreFactory,
-})
-export class FirebaseFirestore {
-    static readonly Blob = firebase.firestore.Blob;
```

Comment:
We removed adding all sorts of statics from "our" classes. Users of ngx-sherlock now frequently do ES imports from 'firebase/*' modules and can get all these constants there.